txpool/txpool_test.go

```diff
@@ -1,1125 +1,1701 @@
 package txpool
 
 import (
-	"context"
-	"crypto/ecdsa"
 	"crypto/rand"
-	"fmt"
 	"math/big"
-	"strconv"
-	"sync"
 	"testing"
+	"time"
 
 	"github.com/0xPolygon/polygon-sdk/chain"
 	"github.com/0xPolygon/polygon-sdk/crypto"
 	"github.com/0xPolygon/polygon-sdk/helper/tests"
-	"github.com/0xPolygon/polygon-sdk/network"
-	"github.com/0xPolygon/polygon-sdk/txpool/proto"
 	"github.com/0xPolygon/polygon-sdk/types"
 	"github.com/hashicorp/go-hclog"
 	"github.com/stretchr/testify/assert"
 )
 
-var forks = &chain.Forks{
-	Homestead: chain.NewFork(0),
-	Istanbul:  chain.NewFork(0),
-}
-
 const (
 	defaultPriceLimit uint64 = 1
 	defaultMaxSlots   uint64 = 4096
+	validGasLimit     uint64 = 10000000
```

Comment:
Is this arbitrary number because this is not valid in term of block gas limit validation. Default value for genesis block gas limit size is 4712388 so we need to put value that si smaller than this but bigger then intrinsicGas

Reply:
I will resolve this in my PR which is based to this branch. https://github.com/0xPolygon/polygon-sdk/pull/327/files